src/value.h

```diff
@@ -177,6 +177,12 @@ class Tensor: public ShapedValue {
       const std::vector<smt::Expr> &dilations,
       ConvLayout layout) const;
 
+  // tensor which is depthwise convolution of this 2D tensor and filter.
```

Comment:
Please leave a comment saying that callers of this fn must check whether the tensors are initialized (look `conv()` above).